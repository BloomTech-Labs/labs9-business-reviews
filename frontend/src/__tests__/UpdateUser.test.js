import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import UpdateUserComponent from '../components/UpdateUser';

describe('UpdateUserComponent />', () => {
  it('renders', () => {
    shallow(<UpdateUserComponent />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<UpdateUserComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
  it('should contain NavBar', () => {
    let app = shallow(<UpdateUserComponent />);
    expect(app.find("NavBar").length).toBe(1);
  });
});
