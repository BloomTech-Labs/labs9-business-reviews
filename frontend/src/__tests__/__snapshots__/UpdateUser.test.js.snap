// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpdateUserComponent /> matches snapshot 1`] = `
<div>
  <NavBar />
  <styled.div>
    <GatedSignIn>
      <withRouter(UserUpdateForm) />
    </GatedSignIn>
  </styled.div>
</div>
`;
